pkg/autodetect/main_test.go

```diff
@@ -88,7 +85,7 @@ func TestStartContinuesInBackground(t *testing.T) {
 	select {
 	case <-done:
 		assert.False(t, viper.GetBool("auth-delegator-available"))
-	case <-time.After(5 * time.Second):
+	case <-time.After(1 * time.Second):
```

Comment:
Why this change?

Reply:
The test is now more resilient. We don't need to wait all that time.